NorsokSCDLibrary.aml

```diff
@@ -25889,13 +25969,18 @@
         <Value />
       </Attribute>
       <Attribute Name="NoteRef" AttributeDataType="xs:string">
-        <Description>Reference to Drawing Note</Description>
+        <Description>Reference to Drawing Note object GUID</Description>
         <DefaultValue />
         <Value />
+        <Constraint Name="aml-RegExp">
+          <UnknownType>
+            <Requirements>^[0-9A-Fa-f]{8}(-[0-9A-Fa-f]{4}){4}[0-9A-Fa-f]{8}$</Requirements>
```

Comment:
Is there a way to factor out the regex instead of copy-paste? I assume it's not expected to change though.

Reply:
AttributeTypeLib was added in this PR to the library (currently for "Parameter" attribute type).
Attribute types could be used for removal of constraints copy-paste.
This will be evaluated (also, if attribute types would be useful for other attributes).

Reply:
Created #54 to track the follow up of this evaluation.